05-JWT-Basics/preferred/public/browser-app.js

```diff
@@ -0,0 +1,70 @@
+const formDOM = document.querySelector('.form')
+const usernameInputDOM = document.querySelector('.username-input')
+const passwordInputDOM = document.querySelector('.password-input')
+const formAlertDOM = document.querySelector('.form-alert')
+const resultDOM = document.querySelector('.result')
+const btnDOM = document.querySelector('#data')
+const tokenDOM = document.querySelector('.token')
+
+formDOM.addEventListener('submit', async (e) => {
+  formAlertDOM.classList.remove('text-success')
+  tokenDOM.classList.remove('text-success')
+
+  e.preventDefault()
+  const username = usernameInputDOM.value
+  const password = passwordInputDOM.value
+
+  try {
+    const { data } = await axios.post('/api/v1/login', { username, password })
```

Comment:
This route doesn't exist in the preferred directory.

Reply:
Fixed it